lib/config/pagy.rb

```diff
@@ -37,6 +37,9 @@
 
 # Backend Extras
 
+# Arel extra: For better performance utilizing grouped ActiveRecord collections:
+# require 'pagy/extras/arel'
```

Comment:
While at it, please, add the reference to the do
```suggestion
# See https://ddnexus.github.io/pagy/docs/extras/array
```

Reply:
Good point. I missed that.

Reply:
Fixed with: [349b15e](https://github.com/ddnexus/pagy/pull/462/commits/349b15eaf868386db660f81e38fbba4458cde7e5)